README.md

```diff
@@ -70,12 +70,22 @@ Service method json format:
 ```json
 [
   {
-    "your_message_field_1": "<value>",
-    "your_message_field_2": "<value>",
+    "input": {
```

Comment:
Maybe input/output should also be rename to something more meaningful?

Reply:
well either request/response or input/output, I can't think of anything better 😐

Reply:
Okay, lets leave it as input/output for now?

Reply:
👍